### ISSUE TITLE
How to input labels for training?

### ISSUE THREAD

**Issue by aayushs879** [eriklindernoren/PyTorch-YOLOv3#120]
How to we feed the input target labels to the forward function. I wanna train this model for Pascal VOC data and i am having trouble figuring out, in what format should we feed the ground truth targets.

**Comment by fourth-archive** [comment]
This repo has a very well explained guide with your format questions answered:
https://github.com/ultralytics/yolov3/wiki/Example:-Train-Single-Class

**Comment by eriklindernoren** [comment]
I haven't tested this PR yet but you can check out #50, where they have implemented a Pascal VOC dataloader for this implementation. You can also see my [instructions](https://github.com/eriklindernoren/PyTorch-YOLOv3#train-on-custom-dataset) for how to train the model on a custom dataset, where I briefly describe how the default dataloader expects the coordinates of bounding boxes to be scales (`[0, 1]`).
